unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Catalog/FavoritesController.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FavoritesController : MonoBehaviour
```

Comment:
`FavoritesController` and `QuickbarController` are having a mixed responsibility of view and controller. If we are going to refactor this to match our project-wide HUD patterns, we should find a way to decouple the UI logic from the presentation logic. 

This decoupling is important to make better tests. If we decouple the view, then the logic of the HUD element is easier to test because it wouldn't be a `MonoBehavior`.

Reply:
I agree with Brian. The controller should just handle the array of items, update their `isFavorite` state and call an update for the view (or generate events for the view to listen, I like both approaches).

It would be perfect if the View is either mockable or not even needed, so we can test it in an editor test suite without the need of playmode.